src/components/ProfileForm/components/ProfileFormStep2.tsx

```diff
@@ -0,0 +1,89 @@
+              <span className="pl-1"> Yes </span>
+            </label>
+            <label>
+              <input type="radio" name="do_you_wear_makeup" value="no" />
+              <span className="pl-1"> No </span>
+            </label>
+          </div>
+        </fieldset>
+        <fieldset className="mb-2 flex flex-col">
+          <legend className="mb-2 font-bold"> What products you use? </legend>
+          <label>
+            <input name="useproducts" value="foundation" type="checkbox" />
+            <span className="pl-1"> Foundation </span>
+          </label>
+          <label>
+            <input name="useproducts" value="concelear" type="checkbox" />
+            <span className="pl-1"> Concelear </span>
+          </label>
+          <label>
+            <input name="useproducts" value="blush" type="checkbox" />
+            <span className="pl-1"> Blush </span>
+          </label>
+          <label>
+            <input name="useproducts" value="bronzer" type="checkbox" />
+            <span className="pl-1"> Bronzer </span>
+          </label>
+          <label>
+            <input name="useproducts" value="contour" type="checkbox" />
+            <span className="pl-1"> Contour </span>
+          </label>
+          <label>
+            <input name="useproducts" value="eyeliner" type="checkbox" />
+            <span className="pl-1"> Eyeliner </span>
+          </label>
+          <label>
+            <input name="useproducts" value="mascara" type="checkbox" />
+            <span className="pl-1"> Mascara </span>
+          </label>
+          <label>
+            <input name="useproducts" value="eyeshadow" type="checkbox" />
+            <span className="pl-1"> Eyeshadow </span>
+          </label>
+          <label>
+            <input name="useproducts" value="lipstick" type="checkbox" />
+            <span className="pl-1"> Lipstick </span>
+          </label>
+          <label>
+            <input name="useproducts" value="lipliner" type="checkbox" />
+            <span className="pl-1"> Lipliner </span>
+          </label>
+          <label>
+            <input name="useproducts" value="powder" type="checkbox" />
+            <span className="pl-1"> Powder </span>
+          </label>
+          <label>
+            <input name="useproducts" value="highlighter" type="checkbox" />
+            <span className="pl-1"> Highlighter </span>
+          </label>
+          <label>
+            <input name="useproducts" value="primer" type="checkbox" />
+            <span className="pl-1"> Primer </span>
+          </label>
+          <label>
+            <input name="useproducts" value="eyebrow" type="checkbox" />
+            <span className="pl-1"> Eyebrow products (pencil, gel, ect) </span>
+          </label>
+          <label>
+            <input name="useproducts" value="other" type="checkbox" />
+            <span className="pl-1"> Other </span>
+            <input name="useproducts" type="text" className="border" />
```

Comment:
Cannot have 2 inputs with the same name unless they are `radio` buttons and we want to only select one.